cd.yml

```diff
@@ -3,14 +3,9 @@ context: identity
 triggering: multibranch
 
 defaultNodes:
-  default: id-small-slave
+  default: id-mite-slave
```

Comment:
We going back to our own slave?

Reply:
We never used our own slave in this repo, AFAIK.  We got it set up for id-mite-nft, but never used it here.  It works on id-small-slave, so this change isn't completely necessary -- but it makes things easier for us (e.g. if we want to make sure we're always on the latest point release of python for running the unit tests)